### PR TITLE
Refactor: utilize new consolidated syntax

### PR DESCRIPTION
**Pull request by TravonteD**

### CODE REVIEW

File: queries/css/highlights.scm [hunk 1]
```diff
@@ -1,72 +1,88 @@
...
+ (attribute_name)
+ ] @property
 
 
 ((property_name) @type
- (#match? @type "^--"))
+                 (#match? @type "^--"))
 ((plain_value) @type
- (#match? @type "^--"))
+               (#match? @type "^--"))
```
Comment (kyazdani42) [inline queries/css/highlights.scm:61]: are those tabs required ? i dont mind but it looks weird on github

Comment (TravonteD) [reply queries/css/highlights.scm:61]: Good catch, it was just vim trying to be smart with the indentation :smile:

File: queries/ruby/highlights.scm [hunk 2]
```diff
@@ -39,32 +48,45 @@
...
+   ])
+
 (method_call
-  method: (constant) @function)
+  [
+   receiver: (constant) @constant
+   method: [
+            (identifier) @function
+            (constant) @function
+            ]
```
Comment (vigoux) [inline queries/ruby/highlights.scm:65]: Isn't this refectorable even further in this ?
```suggestion
   method: [
            (identifier)
            (constant)
            ] @function
```

Comment (TravonteD) [reply queries/ruby/highlights.scm]: I did a quick test and yes that works! I've updated accordingly

**Comment by vigoux** [comment]
That's all good for me, your queries are really an example though ! Can you add that to the `CONTRIBUTING.md` ? Something saying that the reference in term of complicated but accurate queries are the one you maintain

**Comment by TravonteD** [comment]
Added a line to let new contributers know that they can use the ruby queries as an example.

File: queries/css/highlights.scm [hunk 3]
```diff
@@ -1,72 +1,88 @@
...
-"{" @punctuation.bracket
-")" @punctuation.bracket
-"(" @punctuation.bracket
-"}" @punctuation.bracket
+[
+ "#"
+ ","
+ "."
+ ":"
+ "::"
```
Comment (vigoux) [inline queries/css/highlights.scm:80]: Just noticed that you have lost the `@punctuation.delimiter` here

Comment (TravonteD) [reply queries/css/highlights.scm:80]: Updated with a fix

**Pull request merged**